test/e2e/operator_test.go

```diff
@@ -238,7 +238,7 @@ func TestDNSLogging(t *testing.T) {
 	// Verify that the Corefile of DNS DaemonSet pods have been updated.
 	dnsDaemonSet := &appsv1.DaemonSet{}
 	if err := cl.Get(context.TODO(), operatorcontroller.DNSDaemonSetName(defaultDNS), dnsDaemonSet); err != nil {
-		_ = fmt.Errorf("failed to get daemonset %s/%s: %v", dnsDaemonSet.Namespace, dnsDaemonSet.Name, err)
+		t.Errorf("failed to get daemonset %s/%s: %v", dnsDaemonSet.Namespace, dnsDaemonSet.Name, err)
```

Comment:
fatal?

Reply:
Done in https://github.com/openshift/cluster-dns-operator/pull/365/commits/440ae76b555bd3a5dc774158a10e8b8e431903d7.

Reply:
Still shows `t.Errorf` for me, am I missing something?